packages/playout-gateway/src/atemUploader.ts

```diff
@@ -5,6 +5,9 @@ import * as fs from 'fs'
 import { AtemMediaPoolAsset, AtemMediaPoolType } from 'timeline-state-resolver'
 import * as _ from 'underscore'
 import * as path from 'path'
+import { logger } from './logger'
+
+const atemUploaderLogger = logger.tag('atemUploader')
```

Comment:
What's stopping us from just naming this variable `logger` as well?

It's going to be difficult to remember the name of the logger depending of what file you are in. It's much easier to remember that we just need to call `logger`